### ISSUE TITLE
Windows 8.1 gvim winpos inconsistency

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vim/vim#214]
```
Problem setting winpos via _vimrc:
1. Set Windows taskbar to sit along the top of the screen.
2. Set in _vimrc to have "winpos 0 0"
3. Start gvim

The GVim window should sit along the top-left corner, snug against the taskbar. 
But now while it sits flush against the left correctly (xoff is working), it 
sits away from the taskbar by N pixels where N=(height of the taskbar).

Problem setting winpos inside gVim:
1. Set Windows taskbar to sit along the top of the screen.
2. Start gvim
3. :winpos 0 0

The GVim window should sit against the top-left corner but just below the 
taskbar. Instead, it now sits against the absolute top-left corner of the 
screen, meaning GVim's title bar is covered up by the Windows taskbar.

In conclusion:
1. Using winpos from the _vimrc causes gvim to compensate for the taskbar twice.
2. Using winpos from within gvim causes gvim to ignore the existence of the 
taskbar.

The version used is:

VIM - Vi Improved 7.4 (2013 Aug 10, compiled Aug 10 2013 14:38:33)
MS-Windows 32-bit GUI version with OLE support
Compiled by mool@tororo

OS: Windows 8.1
```

Original issue reported on code.google.com by `zxcvf...@gmail.com` on 5 Apr 2014 at 6:11
